web/i18n/defaults/active.en.toml

```diff
@@ -1,5 +1,8 @@
 # default localiztion file for english
 
+# the name of this language, in its language. Used for language picking
+LanguageName = "English"
```

Comment:
this is new key `LanguageName` is used for populating the language picker. it would be a good idea to document it alongside other instructions for adding translations to the existing room